DoublyLinkedList/doubly_linked_list.h

```diff
@@ -0,0 +1,193 @@
+#include <iostream>
+#include <stdio.h>
+using namespace std;
+// Node Creation
+class doubly_linked_list
+{ //class
+        int data;
+        doubly_linked_list *next;
```

Comment:
why is a doubly_linked_list a member of doubly_linked_list?

Reply:
to implement node pointer sir

Reply:
i think we are mixing two things here. 

1. there is the list
2. there are nodes within list. 

why is doubly_linked_list both the list and the node ? they should be separate.